src/Api/PubnubApi/Security/Crypto/Cryptor.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PubnubApi.Security.Crypto
+{
+    public class Cryptor
```

Comment:
What is the purpose of this wrapper?

Reply:
this can be removed.  I created this before CryptoModue.

Reply:
deleted this file